### PR TITLE
[CLI] adding 'superset flower' command (flower is a UI for Celery)

### PR DESCRIPTION
**Pull request by mistercrunch**
Also:
* fixing Celery to truly use the configuration object (was using only one param)
* adding some color to the CLI
screen shot 2017-06-13 at 10 05 41 pm

### CODE REVIEW

File: superset/cli.py [hunk 1]
```diff
@@ -21,6 +22,12 @@
 manager.add_command('db', MigrateCommand)
 
 
+def get_celery_app():
+    app = celery.current_app._get_current_object()
```
Comment (timfeirg) [inline superset/cli.py]: hey quick question, isn't it safe to just use `current_app` here?

Comment (mistercrunch) [inline superset/cli.py]: I'm not sure what you mean

Comment (timfeirg) [inline superset/cli.py]:
```python
    app = celery.current_app
    app.config_from_object(...)
    return app
```
like this maybe?

**Comment by yolken** [comment]
LGTM

**Pull request merged**